Automattic/wp-calypso
# Concierge: Remove available_times endpoint code

The changes in #25916 made it so available 1:1 times were retrieved via the new `initial` endpoint. As a result the `available_times` code is no longer needed. This PR finishes off the migration to the new endpoint and then removes the now un-used `available_times` code.

Removing unused code will be beneficial long term:

- It removes any confusion that may be caused by someone trying to work out where the code is used and what it's for.
- Removes the effort required to maintain the code.
- Eliminates the risk of the code accidentally being reused later and introducing bugs.

**Testing**
As per #25916:

1. Navigate to `/me/concierge` - `/concierge/schedules/{schedule id}/initial` should be requested and `/concierge/schedules/{schedule id}/available_times` should not.
2. Click through to the second booking step, available sessions should be displayed as normal.

Additionally, you can run `npm run test-client concierge` to ensure nothing has been broken.
